pypowsybl/network/impl/pandapower_converter.py

```diff
@@ -0,0 +1,299 @@
+import math
+import typing
+from enum import Enum
+from importlib import util
+from typing import Dict
+
+import numpy as np
+import pandas as pd
+import pypowsybl._pypowsybl as _pp
+from pandapower import pandapowerNet
+from pandas import Series
+from pandas import DataFrame
+
+from .network import Network
+from .network_creation_util import create_empty
+
+MIN_TARGET_P_TO_NOT_BEEN_DISCADED_FROM_ACTIVE_POWER_CONTROL = 0.0001
```

Comment:
```suggestion
MIN_TARGET_P_TO_NOT_BE_DISCARDED_FROM_ACTIVE_POWER_CONTROL = 0.0001
```

Reply:
Fixed

Reply:
you missed the `R`, see https://github.com/powsybl/pypowsybl/pull/786#discussion_r1806422490